test/evaluation/test_evaluator.py

```diff
@@ -417,7 +417,31 @@ def test_MASE_sMAPE_M4(timeseries, res):
 )
 def test_metrics(timeseries, res, has_nans, input_type):
     ts_datastructure = pd.Series
-    evaluator = Evaluator(quantiles=QUANTILES)
+    evaluator = Evaluator(quantiles=QUANTILES, num_workers=0)
+    agg_metrics, item_metrics = calculate_metrics(
+        timeseries,
+        evaluator,
+        ts_datastructure,
+        has_nans=has_nans,
+        input_type=input_type,
+    )
+
+    for metric, score in agg_metrics.items():
+        if metric in res.keys():
+            assert abs(score - res[metric]) < 0.001, (
+                "Scores for the metric {} do not match: \nexpected: {} "
+                "\nobtained: {}".format(metric, res[metric], score)
+            )
+
+
+@pytest.mark.parametrize(
+    "timeseries, res, has_nans, input_type",
+    zip(TIMESERIES, RES, HAS_NANS, INPUT_TYPE),
+)
+def test_metrics(timeseries, res, has_nans, input_type):
```

Comment:
This test function and the above one have the same name: I'm not sure how that will work out, but for clarity you could rename them as `test_metrics_single_processing` and `test_metrics_multi_processing`, or something like that

Reply:
or, renamed it